exercises/hello_hapi/problem.fr.md

```diff
@@ -0,0 +1,51 @@
+Créez un serveur hapi qui écoute sur le numéro de port passé dans
+la ligne de commande, et répond par « Bonjour Hapi » aux requêtes
+HTTP GET envoyées sur `/`.
+
+L’atelier enverra des requêtes à votre serveur et vérifiera leur réponse.
```

Comment:
Because commenting French is easier in French, the following comments will be in French (If you are only English speaker and have question fell free to ask in English)

Dans ce context, j'aurai traduit "workshop" par "exercice". En effet, je pense que le mot "workshop" est employé dans le sens "un exercice réalisé avec workshoper". En français, un atelier ne peut rien envoyer. C'est trop bizarre comme tournure. Dans le contexte d'un exercice, c'est déjà beaucoup plus compréhensible. Eventuellement une périphrase peut aider à rendre les choses plus claire: "L'infrastructure de cette atelier enverra des requêtes au serveur que vous allez écrire et vérifiera la qualité des réponses renvoyées."